caddyconfig/httpcaddyfile/tlsapp.go

```diff
@@ -490,8 +495,9 @@ func newBaseAutomationPolicy(options map[string]any, warnings []caddyconfig.Warn
 	_, hasLocalCerts := options["local_certs"]
 	keyType, hasKeyType := options["key_type"]
 	ocspStapling, hasOCSPStapling := options["ocsp_stapling"]
+	reusePrivateKeys, hasReusePrivateKeys := options["reuse_private_keys"]
```

Comment:
This is looking for a global option called `reuse_private_keys`.

Not necessarily harmful, but I bet if you remove this your `tls` block config will still work...

Reply:
Oh yeah makes sense but it currently doesn't work for global because there's no:
```go
RegisterGlobalOption("reuse_private_keys", parseOptTrue)
```

Would you like me to (add this line and adapt test) or (remove this line)?

Reply:
I'm in favor of making this feature the _least_ accessible possible :wink: 

So let's not add that option, and just leave it for the `tls` directive. And we can probably remove this line (and other lines that rely on it).

Reply:
I kinda wish we had grouped all of these under a `tls` grouping in global options. But too late for that now. I think we should add this though, for completion's sake. And anyways someone will end up complaining about it and requesting this I'm sure :joy:

Reply:
It's just that I intend to remove this later. I'd really like to keep it out of the global options.

Reply:
Done, global stuff is gone.